packages/lesswrong/components/localGroups/SetPersonalMapLocationDialog.tsx

```diff
@@ -62,16 +62,16 @@ const SetPersonalMapLocationDialog = ({ onClose, classes }: {
             initialValue={label}
           /> : <Loading/>}
         </div>
-        <TextField
-            label={`Description${isEAForum ? '' : ' (Make sure to mention whether you want to organize events)'}`}
+        {!isEAForum && <TextField
+            label={`Description (Make sure to mention whether you want to organize events)}`}
```

Comment:
I figured that people would prefer to just fill in their bio once and have that displayed on the map, rather than have two separate text fields (bio and mapText) that are pretty similar, purpose-wise. Let me know if you disagree.